packages/dd-trace/src/util.js

```diff
@@ -20,8 +20,50 @@ function isError (value) {
   return false
 }
 
+// Matches a glob pattern to a given subject string
+function globMatch (pattern, subject) {
```

Comment:
Is this actually faster than something like `wildcard-match` or `picomatch`? A quick benchmark seems to indicate that it would be faster to simply convert the pattern to a regular expression and then testing the regexp.

Reply:
No idea. I just left that as is when I picked up the PR. Didn't consider using some existing module though due to our push to move _away_ from external dependencies.

Reply:
We could also convert manually to a regular expression. I would be fine with either that or an external dependency, since in this case it would be an extremely small and well defined dependency with no transitive dependencies and pretty much 0 risk of issues. Basically the idea is to reduce code to avoid weird bugs, which both approaches would achieve.